SimCalorimetry/EcalSelectiveReadoutAlgos/src/EcalSelectiveReadoutSuppressor.cc

```diff
@@ -308,7 +308,7 @@ EcalSelectiveReadoutSuppressor::run(const edm::EventSetup& eventSetup,
     for(EBDigiCollection::const_iterator digiItr = barrelDigis.begin();
 	digiItr != barrelDigis.end(); ++digiItr){
       int interestLevel
-	= ecalSelectiveReadout->getCrystalInterest(EBDigiCollection::DetId(digiItr->id())) && ~EcalSelectiveReadout::FORCED_MASK;
+	= ecalSelectiveReadout->getCrystalInterest(EBDigiCollection::DetId(digiItr->id())) & ~EcalSelectiveReadout::FORCED_MASK;
```

Comment:
This changes results and should be vetted by simulation.

Reply:
@Dr15Jones indeed this is the least innocent part of the fix, in the limited test I did (wf 10824.0) interestLevel was unchanged when printed out, and no effect can be noticed in the DQM tests (SelectiveReadout should impact the ECAL response and everything that depends on it).  I consider that this will be part of the possible more extensive checks done in RelVals, @civanch who has signed this PR may comment further

Reply:
@Dr15Jones to be more precise: there were some limited changes in the printout, but still no effect was visible in the validation. The final outcome of this needs to be tested with higher statistics